datasets/sources.bib

```diff
@@ -36467,3 +36467,799 @@ @article{zwernemann1963
     year = {1963},
     key = {Zwernemann (1963)}
 }
+@book{-169,
+    author = {Pas},
+    bibkey = {}
+}
+@book{-184,
+    author = {M},
+    bibkey = {}
+}
+@book{-257,
+    author = {Bu},
+    bibkey = {}
+}
```

Comment:
@kirbykat These first three records are artifacts, due to unparseable comment fields and must be removed. Will do so later today.